app/gui2/src/components/GraphEditor/widgets/WidgetSelection.vue

```diff
@@ -136,11 +139,21 @@ watch(selectedIndex, (_index) => {
 </script>
 
 <script lang="ts">
+function hasBooleanTagValues(parameter: SuggestionEntryArgument): boolean {
+  if (parameter.tagValues == null) return false
+  if (parameter.tagValues.length != 2) return false
+  const [first, second] = Array.from(parameter.tagValues).sort()
+  if (first === 'Boolean.False' && second === 'Boolean.True') return true
+  return false
+}
+
 export const widgetDefinition = defineWidget(WidgetInput.isAstOrPlaceholder, {
   priority: 50,
   score: (props) => {
     if (props.input.dynamicConfig?.kind === 'Single_Choice') return Score.Perfect
-    if (props.input[ArgumentInfoKey]?.info?.tagValues != null) return Score.Perfect
+    // Boolean arguments also have tag values, but the checkbox widget should handle them.
+    if (props.input[ArgumentInfoKey]?.info?.tagValues != null && !hasBooleanTagValues)
```

Comment:
Here the `hasBooleanTagValues` function is not actually called.

Reply:
Can we enable some lint for checking it?